ps.go

```diff
@@ -51,8 +51,10 @@ var psCommand = cli.Command{
 			psArgs = []string{"-ef"}
 		}
 
-		output, err := exec.Command("ps", psArgs...).Output()
+		cmd := exec.Command("ps", psArgs...)
+		output, err := cmd.CombinedOutput()
 		if err != nil {
+			fmt.Println(fmt.Sprint(err) + ": " + string(output))
```

Comment:
Can you change this to:

`return fmt.Errorf("%s: %s", err, output)`

This should give you the same results without having to do a println and returns the error with a different message and have duplicated lines